src/concrete/ml/quantization/quantizers.py

```diff
@@ -369,7 +369,7 @@ def check_is_uniform_quantized(self, options: QuantizationOptions) -> bool:
 
         re_quant_scales = numpy.rint(unique_scales / min_scale) * min_scale
 
-        return numpy.allclose(unique_scales, re_quant_scales, atol=0.02)
+        return utils.check_allclose_and_shape(unique_scales, re_quant_scales, atol=0.02)
```

Comment:
I think you can directly change to `from utils import ...` and call the function directly, it's more readable !